pkgs/development/compilers/solc/default.nix

```diff
@@ -7,21 +7,21 @@ assert z3Support -> z3 != null && stdenv.lib.versionAtLeast z3.version "4.6.0";
 assert cvc4Support -> cvc4 != null && cln != null && gmp != null;
 
 let
-  jsoncppURL = "https://github.com/open-source-parsers/jsoncpp/archive/1.9.2.tar.gz";
+  jsoncppURL = "https://github.com/open-source-parsers/jsoncpp/archive/1.9.3.tar.gz";
   jsoncpp = fetchzip {
     url = jsoncppURL;
```

Comment:
```suggestion
  jsoncppVersion = "1.9.3";
  jsoncpp = fetchzip {
    url = "https://github.com/open-source-parsers/jsoncpp/archive/${jsoncppVersion }.tar.gz";
```

Reply:
The jsoncppURL is used in a replace statement further in the recipe, but indeed it's good to factor the version out of it, and while I was at it, I updated to 1.9.4 which is the latest release.